bootstrap.sh

```diff
@@ -33,6 +33,8 @@ export JUPYTER_PATH=/srv/.jupyter
 export JUPYTER_RUNTIME_DIR=/srv/.local/share/jupyter/runtime
 export JUPYTER_DATA_DIR=/srv/.local/share/jupyter
 export IPYTHONDIR=/srv/.ipython
+export CONDOR_CONFIG=/srv/.condor_config
```

Comment:
I would suggest not using an environment variable that starts with CONDOR to avoid potential conflicts. Maybe prepend it with something? (like COFFEA_CONDOR_CONFIG)

Reply:
`CONDOR_CONFIG` is required for a "custom" configuration settings to be used. The convention for a changing single condor variables/settiongs via environment variables would be `_CONDOR_<VAR>`

https://htcondor.readthedocs.io/en/latest/admin-manual/introduction-to-configuration.html#ordered-evaluation-to-set-the-configuration

Reply:
I am quoting an extract from the link you posted:

> For Unix platforms, the location of the single initial configuration file starts at the top of the following list. The first file that exists is used, and then remaining possible file locations from this list become irrelevant.
> 
> 1.     the file specified by the CONDOR_CONFIG environment variable. If there is a problem reading that file, HTCondor will print an error message and exit right away.
> 2.     /etc/condor/condor_config
> 3.     /usr/local/etc/condor_config
> 4.     ~condor/condor_config

I think I get why you are trying to use this environment variable (you want to set this globally to avoid copying the script to /etc/condor/config.d/ or bind mounting it as /etc/condor/condor_config). My point is that you are setting a variable that controls HTCondor behavior and it is tweaked by site administrators and/or tools like glideinWMS in some cases.

At the LPC, it probably doesn't matter though. We do not run HTCondor on interactives, so this variable is meaningless. On our batch workers, we let this environment variable be (there are some that we reset no matter what the user provides/adds). The computing person in me always tries to go for completeness and portability so I'd again suggest not tweaking that environment variable and going with something auxiliary/custom. Ultimately it is up to you guys because you understand your use cases better than I do.

Reply:
Ultimately, this variable is set only within the client container, which is run inside the LPC interactives. The batch jobs don't execute this bashrc.
I am definitely open to other ideas! The only requirement here is that we can submit jobs to the LPC cluster from inside this container.